frontend/src/metabase/query_builder/components/view/ViewHeaderUtils.js

```diff
@@ -0,0 +1,19 @@
+import * as Lib from "metabase-lib";
+
+/**
+ * We can only "explore results" (i.e. create new questions based on this one)
```

Comment:
we use capitalized file naming only for components, utils usually are placed in `utils` file

Reply:
Thanks.
Will follow up soon.